corebuild/dotnet.ps1

```diff
@@ -10,12 +10,11 @@ if ((Test-Path $bootStrapperPath) -eq 0)
         mkdir $toolsLocalPath | Out-Null
     }
 
-    cp "bootstrap.ps1" $bootStrapperPath
+    cp (Join-Path $PSScriptRoot "bootstrap.ps1") $bootStrapperPath
 }
 
-
 # now execute it
-& $bootStrapperPath (Get-Location) $toolsLocalPath | Out-File (Join-Path (Get-Location) "bootstrap.log")
```

Comment:
How does a dev build just one vertical instead of all configurations.  ie, just build netcore_release|x64?  I'm not familiar with your teams product / workflow, so maybe it's not an issue.  Do they just use Visual Studio?